CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - Added option `label_position = :center` to place labels centered over each bar [#4274](https://github.com/MakieOrg/Makie.jl/pull/4274).
 - Fixed Boundserror in clipped multicolor lines in CairoMakie [#4313](https://github.com/MakieOrg/Makie.jl/pull/4313)
 - Fix float precision based assertions error in GLMakie.volume [#4311](https://github.com/MakieOrg/Makie.jl/pull/4311)
+- Added way for `PolarAxis` to pull fontsize attribute from `Figure()` [#4314](https://github.com/MakieOrg/Makie.jl/pull/4314)
```

Comment:
```suggestion
- `PolarAxis` fontsize now defaults to global figure `fontsize` in the absence of specific `Axis` theming [#4314](https://github.com/MakieOrg/Makie.jl/pull/4314)
```

Just to make the verbiage more specific :)

Reply:
Can I resolve conversations like this when making a change that should fix the problems mentioned or is that something usually reserved for the reviewers?

Reply:
Feel free to resolve if the comment is addressed!  If we feel it's not, we can always reopen the comment.

Reply:
Alright, thank you!